src/components/general/OutcomeNodeContent.tsx

```diff
@@ -174,7 +180,7 @@ const OutcomeNodeContent = (props: OutcomeNodeContentProps) => {
   // useEffect(() => console.log('subNodes changed'), [subNodes]);
 
   return (
-    <div>
+    <div role="tabpanel" id={`tabpanel-${node.id}`}>
```

Comment:
I think I used  `aria-labelledby` and `tabIndex` here too as per [this example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example)

Not sure if necessary though

Reply:
I don't think they're necessary in this case:

- `aria-labelledby`: There's a `h4` inside each `tabpanel` which labels the tabpanel content (e.g. "Gebäude")
- `tabIndex`: The tabpanel itself doesn't need to be accessible via keyboard navigation, but the content within it is